packages/dataviews/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- The filters that use `in` and `notIn` operators (the values by default) support now multi-selection. Use `equal` and `notEqual` for single-selection filters. Note that the value in the `filters.view` would be an array for multi-selection as well.
```

Comment:
Thanks for adding to the changelog and it's something we should all start doing. We should also add a header for the type of changes, ex. `Enhancements`, etc..

Reply:
How would you tag this one? It's a "New Feature" but changes the way the existing filters work. I'm tempted to use "Breaking change" to give 3rd parties notice, but it feels too heavy at the same time — nothing would actually break.

Reply:
I'd say enhancement for now.